From 778bb5380f027758d6f839a1d816cc8e39bb58d6 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <aalonso@freescale.com>
Date: Fri, 17 Feb 2012 12:14:28 -0600
Subject: [PATCH 1069/2463] ENGR00171079-3 fsl_devices add generic audio codec
 platform data

* Add generic audio codec platform_data to be able to
  pass specific settings to audio codecs found in Freescale
  target boards.
* cs42888 get this platform data to override specific settings

Signed-off-by: Adrian Alonso <aalonso@freescale.com>
---
 include/linux/fsl_devices.h |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index e96cca4..f6098fb 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -358,6 +358,12 @@ struct mxc_audio_platform_data {
 	void *priv;		/* used by board specific functions */
 };
 
+/* Generic parameters for audio codecs
+ */
+struct mxc_audio_codec_platform_data {
+	int rates; /* codec platform data */
+};
+
 struct mxc_pwm_platform_data {
 	int pwmo_invert;
 	void (*enable_pwm_pad) (void);
-- 
1.7.7.4

